charts/hush-sensor/templates/hushsecrets.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: hush-secrets
```

Comment:
IMO this secret name should not be fixed, to support multiple releases of the chart, each with its own token. In other words, it should be bound to release name. For example see `hush-sensor.imagePullSecretName`.

Reply:
@r-bk what do u mean by multiple releases? Do u mean multiple installation of the chart on the same cluster?
And question: if it won't be fixed doesn't it mean that on helm upgrade the admin will need to reonboard (i.e. create a new secret and pass it here)?

Reply:
@horomena

Yes. Helm allows multiple installations of a chart under different release names. If we use a static name for this secret every new onboard will overwrite the token for all previous releases. If we bind the secret name to release name it is ensured that a new secret will be created per release. We do the same with Snoopy's configmap to allow different configuration per release. This allows a lot of flexibility, as one can deploy different versions of the chart on different pools of nodes.

The admin doesn't need to reonboard. He has the flexibility to use a new token for the new release, or specify an old one. The downside is that if old token is wanted, he must dig the value, either in our UI or using `helm get values <release-name>`.

`helm upgrade` always happens within a specific release. So it will not change the secret name. Admin can reonboard at this stage too if he specifies a new token in `values.yaml`, but if he doesn't specify the Secret value will remain intact and old token will be used.

Reply:
OK thanks for the details.
I don't see a lot of value for supporting installation under different release names, however if given a release all upgrades keep the same configmaps and secrets then I'm completely fine with this approach.

Reply:
Fixed in revision 2